kernels.hcu

```diff
@@ -0,0 +1,5 @@
+#pragma once
```

Comment:
Maybe we can also wrap the kernel launches in in `kernels.cu` and forward declare them when needed? That could save us from adding these header files.

Reply:
That does not help graphs I am afraid. Adding a kernel node requires the function symbol of the actual kernel.
So we would declare them twice in two locations in different ways.